### ISSUE TITLE
Missing pretrain norm_state files of detailed robot embodiments

### ISSUE THREAD

**Issue by Anonymous-694** [Robbyant/lingbot-vla-v2#6]
Hi lingbot team, thanks for your great work. I was wondering if you can opensource your detailed pretrain norm states file of different robot embodiment. Looking forward to your reply.

**Comment by LuFan31** [comment]
Thanks for your interest!

Sorry, but we currently do not plan to release the files associated with our pre-training data, including the corresponding pre-training normalization statistics.

For post-training on your own dataset, you can compute the required `norm_state` / normalization statistics following the instructions in:

`lingbotvla/data/vla_data/README.md`

Please refer specifically to **Section 4: Compute Normalization Statistics**, which provides examples for both single-dataset and multi-dataset settings.
